automates/text_reading/src/main/scala/org/clulab/aske/automates/entities/RuleBasedEntityFinder.scala

```diff
@@ -114,9 +114,9 @@ class RuleBasedEntityFinder(
   private def filterEntities(entities: Seq[Mention]): Seq[Mention] = {
     // ignore citations and remove any entity that is too long given our criteria
     val filteredEntities = entities.filter(m => EntityConstraints.withinMaxLength(m, maxLength))
-    val longest = RuleBasedEntityFinder.keepLongest(filteredEntities, new State())
+//    val longest = RuleBasedEntityFinder.keepLongest(filteredEntities, new State())
```

Comment:
this is commented out to extract smaller phrases, right?

Reply:
Yes, guess I've mistaken the branch again. Thanks for catching that!